bin/install-package-tests

```diff
@@ -47,6 +47,12 @@ if [ -n "${WP_CLI_TEST_DBPASS}" ]; then
   TEST_PASSWORD="${WP_CLI_TEST_DBPASS}"
 fi
 
+while ! mysql --user="${USER}" "${PASSWORD_STRING}" --execute="SHOW DATABASES;" | grep 'information_schema' >/dev/null;
+do
+  echo 'Waiting for MySQL query...'
+  sleep 1
+done
```

Comment:
1. This will be complicated in this variant.
2. GHA already cancels jobs alter definable timeout - 6 hours by default AFAIK.

Reply:
> This will be complicated in this variant.

Sounds good. We can try this, then.

> GHA already cancels jobs alter definable timeout - 6 hours by default AFAIK.

Seems a bit wasteful to wait that long. I added a 3 minute timeout.